src/browser/ui/__tests__/ReactMount-test.js

```diff
@@ -144,4 +144,13 @@ describe('ReactMount', function() {
     ReactMount.render(<div />, container);
     expect(console.warn.mock.calls.length).toBe(0);
   });
+
+  it('should warn when mounting into document.body', function () {
+    spyOn(console, 'warn')
+    ReactMount.render(<div />, document.body);
```

Comment:
This actually blows away the test runner UI in debug mode (`grunt test --debug`, then open http://127.0.0.1:9999/test/). Maybe we can make an iframe to render into and then instead check the tag name of the container instead of for raw equality to `document.body`?